.github/CODEOWNERS

```diff
@@ -1,110 +1,77 @@
 * @bader
 
 # Front-end compiler
-clang/ @premanandrao @elizabethandrews @smanna12
-
-# Driver
-clang/**/Driver @mdtoguchi @AGindinson @hchilama
+clang/ @dpcpp-cfe-reviewers
 
 # LLVM-SPIRV translator
-llvm-spirv/ @AlexeySotkin @AlexeySachkov
+llvm-spirv/ @intel/dpcpp-spirv-reviewers
```

Comment:
Do we have all the teams referenced in this file?

Reply:
We will have to create them once all prior code owners agreed on the change. Will ping them soon.